index.html

```diff
@@ -1,115 +1,276 @@
-                        We are a heavy roleplaying server for the game 'Space Station 13' on the <a href="http://www.byond.com/">BYOND </a> platform where you play as an employee of the trans-stellar corporation known as Nanotrasen, working on one of their research stations posted on the fringes of colonized space.
-                        <br/>
-                        <br/> The frontier is a dangerous place, however, and Nanotrasen has made many enemies on their rise to power. Spies, saboteurs, hostile aliens, dangerous computer programs, and people with seemingly supernatural abilities. Lax hiring standards mean accidents are commonplace, and it's a very thin barrier that holds the uncaring, airless space at bay.
-                        <br />
-                        <br/> So who will you be? An engineer, tasked to keep the station powered and intact enough to be hospitable; a scientist who advances human knowledge about alien artifacts or advanced technologies; the station's artificial intelligence, overseeing everything to serve and protect, or one of the dastardly antagonists? Come on over and find out.
-                        <br />
-                    </div>
-                </div>
-            </div>
-            <div class="ui one column grid centered">
-                <div class="copyright">Copyright Baystation12 2015. Space tiles by Goonstation / <a class="anocolor" href="https://creativecommons.org/licenses/by-sa/2.0/"> CC BY SA 2.0 </a> </div>
-            </div>
-        </div>
-    </div>
-</body>
-
-</html>
+			body {
+				width: 100%;
+				height: 100%;
+				overflow: hidden;
+				background: #171717;
+				background-size: 560px auto;
+				background-image: url("../images/bg.png");
+				animation: bgScroll 15s linear infinite;
+			}
+			
+			#monkey {
+				z-index: -1;
+				min-width: 128px;
+				min-height: 128px;
+				position: absolute;
+				background-image: url("../images/monkey.png");
+				animation: monkeyMove 48s linear infinite;
+			}
+			
+			#meteor1 {
+				z-index: -1;
+				min-width: 128px;
+				min-height: 128px;
+				position: absolute;
+				background-image: url("../images/l_meteor.gif");
+				animation: meteor1Move 33s linear infinite;
+			}
+			
+			#meteor2 {
+				z-index: -1;
+				min-width: 128px;
+				min-height: 128px;
+				position: absolute;
+				background-image: url("../images/s_meteor.gif");
+				animation: meteor2Move 29s linear infinite;
+			}
+			
+			#syndicate1 {
+				z-index: -1;
+				min-width: 96px;
+				min-height: 384px;
+				position: absolute;
+				background-size: 96px auto;
+				background-image: url("../images/syndicate.png");
+				animation: syndicate1Move 50s linear infinite;
+			}
+			
+			#syndicate2 {
+				z-index: -1;
+				min-width: 96px;
+				min-height: 384px;
+				position: absolute;
+				background-size: 96px auto;
+				background-image: url("../images/syndicate.png");
+				animation: syndicate2Move 55s linear infinite;
+			}
+			
+			html {
+				font-size: 14px;
```

Comment:
Use `em` or `%` for fonts for accessibility reasons. Some browsers may adjust font sizes to increase readability, but `px` is fixed size.